src/data/critical-constants-acentric-factor.ts

```diff
@@ -1263,4 +1263,1487 @@ export const DIPPRCriticalConstantsAndAcentricFactorDictionary: Record<
+    CAS: '78-80-8',
+    molecularWeight: 66.10114,
+    criticalTemperature: 492,
+    criticalPressure: 4.38,
+    criticalVolume: 0.248,
+    criticalCompressibilityFactor: 0.266,
+    acentricFactor: 0.137046,
+  },
+
+  MethylbutylEther: {
+    name: 'Methylbutyl ether',
+    formula: 'C5H12O',
+    CAS: '628-28-4',
+    molecularWeight: 88.14818,
+    criticalTemperature: 512.74,
+    criticalPressure: 3.371,
+    criticalVolume: 0.329,
+    criticalCompressibilityFactor: 0.26,
+    acentricFactor: 0.313008,
+  },
+
+  MethylbutylSulfide: {
+    name: 'Methylbutyl sulfide',
+    formula: 'C5H12S',
+    CAS: '628-29-5',
+    molecularWeight: 104.214,
+    criticalTemperature: 593,
+    criticalPressure: 3.47,
+    criticalVolume: 0.36,
+    criticalCompressibilityFactor: 0.253,
+    acentricFactor: 0.3229,
+  },
+
+  '3Methyl1butyne': {
+    name: '3-Methyl-1-butyne',
+    formula: 'C5H8',
+    CAS: '598-23-2',
+    molecularWeight: 68.11702,
+    criticalTemperature: 463.2,
+    criticalPressure: 4.2,
+    criticalVolume: 0.275,
+    criticalCompressibilityFactor: 0.3,
+    acentricFactor: 0.308085,
+  },
+
+  MethylButyrate: {
+    name: 'Methyl butyrate',
+    formula: 'C5H10O2',
+    CAS: '623-42-7',
+    molecularWeight: 102.1317,
+    criticalTemperature: 554.5,
+    criticalPressure: 3.473,
+    criticalVolume: 0.34,
+    criticalCompressibilityFactor: 0.256,
+    acentricFactor: 0.377519,
+  },
+
+  Methylchlorosilane: {
+    name: 'Methylchlorosilane',
+    formula: 'CH5ClSi',
+    CAS: '993-00-0',
+    molecularWeight: 80.5889,
+    criticalTemperature: 442,
+    criticalPressure: 4.17,
+    criticalVolume: 0.246,
+    criticalCompressibilityFactor: 0.279,
+    acentricFactor: 0.225204,
+  },
+
+  Methylcyclohexane: {
+    name: 'Methylcyclohexane',
+    formula: 'C7H14',
+    CAS: '108-87-2',
+    molecularWeight: 98.18606,
+    criticalTemperature: 572.1,
+    criticalPressure: 3.48,
+    criticalVolume: 0.369,
+    criticalCompressibilityFactor: 0.27,
+    acentricFactor: 0.236055,
+  },
+  '1Methylcyclohexanol': {
+    name: '1-Methylcyclohexanol',
+    formula: 'C7H14O',
+    CAS: '590-67-0',
+    molecularWeight: 114.18546,
+    criticalTemperature: 686,
+    criticalPressure: 4,
+    criticalVolume: 0.374,
+    criticalCompressibilityFactor: 0.262,
+    acentricFactor: 0.221299,
+  },
+  cis2Methylcyclohexanol: {
+    name: 'cis-2-Methylcyclohexanol',
+    formula: 'C7H14O',
+    CAS: '7443-70-1',
+    molecularWeight: 114.18546,
+    criticalTemperature: 614,
+    criticalPressure: 3.79,
+    criticalVolume: 0.374,
+    criticalCompressibilityFactor: 0.278,
+    acentricFactor: 0.68049,
+  },
+  trans2Methylcyclohexanol: {
+    name: 'trans-2-Methylcyclohexanol',
+    formula: 'C7H14O',
+    CAS: '7443-52-9',
+    molecularWeight: 114.18546,
+    criticalTemperature: 617,
+    criticalPressure: 3.79,
+    criticalVolume: 0.374,
+    criticalCompressibilityFactor: 0.276,
+    acentricFactor: 0.67904,
+  },
```

Comment:
```suggestion
  },
  
```

Reply:
@qur786 in this case you can just comment once for new line instead of commenting multiple times.